packages/interface/package.json

```
@@ -0,0 +1,37 @@
+{
+	"name": "@wordpress/interface",
+	"version": "0.0.1",
+	"private": true,
+	"description": "Interface module for WordPress.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress"
```

We can add more keywords, e.g.:
```suggestion
		"wordpress",
		"interface",
		"components"
```